pkgs/top-level/python-packages.nix

```diff
@@ -10065,6 +10065,27 @@ in {
     };
   };
 
+  OWSLib = buildPythonPackage rec {
```

Comment:
Could you move this to a separate file?

Reply:
Yes. There is a mix of inline and separate file in python-packages. Is the standard to move to separate file?

Reply:
In the past having module expressions in this file was okay; nowadays, we are moving them to separate files.

Reply:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.